src/routes/donate/+page.svelte

```diff
@@ -0,0 +1,328 @@
+<script lang="ts">
+	import { fly, slide } from 'svelte/transition';
+	import { quintOut } from 'svelte/easing';
+	import Meta from '$lib/components/Meta.svelte';
+	import Footer from '$layout/Footer/FooterHost.svelte';
+	import Button from '$lib/components/Button.svelte';
+	import Snackbar from '$lib/components/Snackbar.svelte';
+	import QRCode from './QRCode.svelte';
+
+	import { queries } from '$data/api';
+	import { createQuery } from '@tanstack/svelte-query';
+	import Query from '$lib/components/Query.svelte';
+	import Dialogue from '$lib/components/Dialogue.svelte';
+
+	let qrCodeDialogue = false;
+	let qrCodeValue = '';
+	let qrCodeDialogueName = '';
+	const teamQuery = createQuery(['team'], queries.team);
+	const donateQuery = createQuery(['donate'], queries.donate);
+	let snackbarOpen = false;
+	let expandCrypto = false;
+
+	async function copyToClipboard(walletAddress: string) {
+		snackbarOpen = true;
+		qrCodeDialogue = false;
+		try {
+			await navigator.clipboard.writeText(walletAddress);
+		} catch (error) {
+			console.error('Failed to copy crypto wallet:', error);
+		}
+	}
+</script>
+
+<Meta title="Donate" />
+
+<main class="wrapper" in:fly={{ y: 10, easing: quintOut, duration: 750 }}>
+	<h2>🎉 Support <span style="color: var(--accent-color);">ReVanced</span></h2>
+	<p>
+		ReVanced offers a variety of patches, including ad-blocking, custom themes, and innovative
+		features. All of which is completely open source and free of charge. Donating will allow
+		ReVanced maintain our servers and develop new features.
+	</p>
+	<Query query={donateQuery} let:data>
+		{#if data.platforms}
+			<div class="buttons-container">
+				{#each data.platforms as platform, i}
+					{#if i === 0}
```

Comment:
The API does not return an ordered list. The API should support a "primary" method which you would use to display the button for it filled.